include/stl2/detail/algorithm/adjacent_find.hpp

```diff
@@ -25,7 +25,7 @@ STL2_OPEN_NAMESPACE {
 	template <ForwardIterator I, Sentinel<I> S, class Pred = equal_to<>,
 		class Proj = identity>
 	requires
-		models::IndirectCallableRelation<
+		models::IndirectInvocableRelation<
```

Comment:
I believe this has been renamed `IndirectRelation`. Ditto for `IndirectPredicate` and `IndirectStrictWeakOrder`.